src/rgw/rgw_op.cc

```diff
@@ -1881,6 +1881,146 @@ void RGWStatBucket::execute()
   }
 }
 
+int RGWGetBucketOplog::verify_permission()
+{
+  if (s->user->user_id.compare(s->bucket_owner.get_id()) != 0)
```

Comment:
for tenant support use:
  if (false == s->auth_identity->is_owner_of(s->bucket_owner.get_id())) {
    return -EACCES;
  }